### ISSUE TITLE
get rid of ./docker-entrypoint.sh

### ISSUE THREAD

**Issue by hairmare** [adfinis/timed-frontend#725]
feel free to do a git blame on the file, it me and i wanna get rid of this tech-dept asap, it does unmentionable things to your deployment that @anehx for sure hates.

https://github.com/adfinis/timed-frontend/blob/ac8dbd89ebb1a316b16e43675261d38a44e02b10/docker-entrypoint.sh#L5-L22

i think i've got a strat to get rid of the file i mention and also set y'all up towards using ghcr and @adfinisbot nicely :tm: 

i'm creating this issue and assigning myself for tracking that i owe -frontend some :heart:  ala how i recently migrated -backend to ghcr :grimacing:

**Comment by anehx** [comment]
Yes please! I'd love to hear and discuss that strategy of yours!

**Comment by hairmare** [comment]
it's the "lets have middleware do it" strat and this now tracking it :trollface:
